ports/kf5codecs/vcpkg.json

```
@@ -0,0 +1,11 @@
+{
+  "name": "kf5codecs",
+  "version-string": "5.75.0",
```

```suggestion
  "version": "5.75.0",
```
Please see https://github.com/microsoft/vcpkg/blob/master/docs/specifications/versioning.md#22-package-versions to get details.